### PR TITLE
Network reimplementation

### PR DESCRIPTION
**Pull request by yardenas**
### Main Ideas:
- [x] Migrate the net to tensorflow 2
- [x] Add dropout and batch normalization
- [x] Remove anchor stuff

Essentially rewrite the net

### CODE REVIEW

**Comment by yardenas** [comment]
Generally it's done but working slower than its Tensorflow 1.13 counterpart
